DM-22039: Rename dax_ppdb to dax_apdb (ap_pipe)

andy-slac opened the pull request:
Renames all PPDB names into APDB.

kfindeisen commented on doc/lsst.ap.pipe/apdb.rst:
```diff
@@ -1,37 +1,37 @@
-|ap_pipe| needs an existing Prompt Products Database (PPDB) in which to store its results.
-Such a database will be provided externally during operations, but developers can run |make_ppdb| to set up their own database for testing.
-This page provides an overview of how to use |make_ppdb|.
+|ap_pipe| needs an existing Prompt Products Database (APDB) in which to store its results.
```
Alert Production Database

kfindeisen commented on doc/lsst.ap.pipe/pipeline-tutorial.rst:
```diff
@@ -112,7 +112,7 @@ The result from running ``ap_pipe`` should look something like
 
-   ppdb/
+   apdb/
       association.db   <--- the Prompt Products Database with DIAObjects
```
Alert Production Database

kfindeisen commented on python/lsst/ap/pipe/make_apdb.py:
```diff
@@ -87,8 +87,8 @@ def parse_args(self, args=None, namespace=None):
-def makePpdb(args=None):
-    """Create a PPDB according to a config.
+def makeApdb(args=None):
+    """Create a APDB according to a config.
```
a -> an

andy-slac commented:
Thanks, I thought I replaced all "Prompt Product Database" but apparently missed some (or all) in this package. I ran another check and fixed them all.
